src/components/MessageList.tsx

```diff
@@ -15,7 +15,7 @@ function MessageList() {
   }, [chatHistory]);
 
   return (
-    <div className="flex-1 overflow-auto" ref={scrollRef}>
```

Comment:
Is it intentional to remove the scrollref here?

Reply:
Must've accidentally removed it, thanks for the reviews and I'll implement the fixes later today